extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/security/FormAuthenticationMechanism.java

```diff
@@ -134,6 +135,8 @@ public void handle(Void event) {
                                 uniEmitter.complete(null);
                                 return;
                             }
+                            exchange.put(FORM_AUTHENTICATION_PARAMETERS, res);
```

Comment:
@sberyozkin the `MultiMap res` is taken from `exchange`, any point of putting it right back and storing it on  the `RoutingContext` twice? Why are not users taking it from `exchange.request().formAttributes()` instead?

Reply:
I suppose that somehow `RoutingContext` attributes are passed somewhere while `RoutingContext` is not? I'll need little more context because https://github.com/quarkusio/quarkus/discussions/38617 doesn't provide use case and it doesn't make sense to me. The `RoutingContext` should be available in the augmentor in the latest versions, shouldn't it?

Reply:
@michalvavrik Sure, I'll check if `exchange.request().formAttributes()` works after the form authentication mechanism reads them.